src/components/Image/ImageLoader.jsx

```diff
@@ -35,8 +50,13 @@ class ImageLoader extends React.Component {
       this.img.onload = this.img.onerror = null
       this.img.src = ''
     }
+    this.unsubscribeRealtime()
   }
 
+  unsubscribeRealtime = () => {
+    this.realtime &&
```

Comment:
Can `this.realtime` be `undefined` at this stage ?

Reply:
I think we can have a race condition between the didMount() and unmount() so I added a check to be sure